install_config/install/docker_registry.adoc

```diff
@@ -239,7 +239,17 @@ this repository via an image.
 To access the registry directly, such as to perform `docker push` or `docker pull` operations, you
 must first log in to the registry using an access token.
 
-. Ensure you are logged in to OpenShift as a link:../../architecture/core_concepts/projects_and_users.html#users[regular user]:
+. Create a link:../../architecture/core_concepts/projects_and_users.html#users[regular user], for example, with `HTPASSWD` authentication:
++
+----
+# htpasswd /etc/openshift/openshift-htpasswd reguser
+----
++
+For more information on `HTPASSWD`, see the
+link:../../admin_guide/configuring_authentication.html#HTPasswdPasswordIdentityProvider[Administrator
+Guide].
```

Comment:
Suggest calling it by the topic name:

s/the Administrator Guide/Configuration Authentication/

Also cuz on docs.openshift.com/org it's currently called "Administration", and we're planning on changing the name soon to "Cluster Administration" or something similar.